Lesson22/assets/js/first.js

```diff
@@ -0,0 +1,11 @@
+
+//Request a three-digit number from a user and check whether it has identical digits in it.
+
+let num=prompt ('Give me a three-digit number, please.');
+console.log(num);
+if (num[0] == num[1] || num[0] == num[2] || num[1] == num[2]){
```

Comment:
this logic is wrong
Check 112 number

Reply:
> this logic is wrong Check 112 number

I checked. It works. What mistake I did?